test_utils/factories.py

```diff
@@ -46,6 +47,7 @@ class Meta(object):
     name = factory.LazyAttribute(lambda x: FAKER.company())
     active = True
     site = factory.SubFactory(SiteFactory)
+    catalog = factory.LazyAttribute(lambda x: random.uniform(1, 1000000))
```

Comment:
FAKER.pyint() could give us a negative int. We could do math.abs on that, but either way, we're moving away from "pure" test methods.